src/gnoi-golang/system-services/traceroutegnoi.go

```diff
@@ -0,0 +1,52 @@
+//as of 4.26.1F Currently not working but shows up in refraction.
```

Comment:
traceroute should be supported. suspect this is vestigial.

Reply:
Replicated this by accident.